Makefile

```diff
@@ -127,11 +127,7 @@ devtest:
 	ghcid --command 'cabal repl integration' --test='Testlib.Run.mainI []'
 
 .PHONY: sanitize-pr
-sanitize-pr:
-	./hack/bin/generate-local-nix-packages.sh
-	make formatf
-	make hlint-inplace-pr
-	make hlint-check-pr  # sometimes inplace has been observed not to do its job very well.
```

Comment:
Why did you change this? Now it will run `formatc` instead of `foramtf` and the linting will be slower.

Reply:
fair point, but i think the change should be in the other direction: [08c94df](https://github.com/wireapp/wire-server/pull/4088/commits/08c94df8645f4e5a973db60efcb7c9b4dc8c6ba4)

Reply:
I don't think it is a good idea. I'd say we should let CI be more thorough and we can keep having a little shortcut for our dev experience.
Because this is becoming a discussion, perhaps we should create another ticket/PR for this.

Reply:
more good points.  give me one last chance: https://github.com/wireapp/wire-server/pull/4088/commits/5a6b9ef168c0a2e0b8bbc25e58c34453e0117cbb